liubai-frontends/liubai-weixin/miniprogram/utils/LiuTime.ts

```
@@ -0,0 +1,37 @@
+
+
+export class LiuTime {
+
+  static diff = 0
+
+  static getLocalTime() {
+    return new Date().getTime()
+  }
+
+  static getTime() {
+    return new Date().getTime() + this.diff
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)